tests/scripts/create-dev-cluster.sh

```diff
@@ -51,24 +53,25 @@ show_ceph_dashboard_info() {
     echo "   API_HOST: $PROMETHEUS_API_HOST"
     fi
     echo "==========================="
+    echo " "
+    echo " *** To start using your rook cluster please set the following alias: "
+    echo " "
+    echo "    > " alias kubectl=\"minikube --profile $ROOK_PROFILE_NAME kubectl --\"
```

Comment:
Why not print the `KUBECTL` variable here to avoid the rewrite?

Reply:
@leseb I like the idea of the MINIKUBE variable. I updated the code to introduce the proposed changes.